lib/iex/lib/iex/introspection.ex

```diff
@@ -141,6 +146,18 @@ defmodule IEx.Introspection do
     end
   end
 
+  defp callback_module(mod, fun, arity) do
+    mod.module_info(:attributes)
+    |> Keyword.get_values(:behaviour)
+    |> Stream.concat()
+    |> Stream.filter(fn(module)->
+      callbacks = module.module_info(:attributes)
```

Comment:
It'd be better to put `module.module_info(:attributes)` on next-line; really not easy to see it's a part of pipeline at first glance.